dev/io.openliberty.jakarta.mail.2.0/.classpath

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
If I am reading this code correctly then this project contains an Open Liberty API, not a Jakarta one. So I think it should be named just `io.openliberty.mail.2.0`

Reply:
@uberskigeek Does this project contain a Jakarta API or an Open Liberty one?